marine.c

```diff
@@ -227,13 +227,62 @@ static GHashTable *packet_filters;
 static int *packet_filter_keys[4096];
 static gboolean prefs_loaded = FALSE;
 
+static marine_pref_t optimize_perfs[] = {
```

Comment:
I think you can disable more reassembly options in IPv4, IPv6 and HTTP (and if you think of more common protocols for this, awesome).